card/client.go

```diff
@@ -129,6 +141,10 @@ func Del(id string, params *stripe.CardParams) (*stripe.Card, error) {
 }
 
 func (c Client) Del(id string, params *stripe.CardParams) (*stripe.Card, error) {
+	if params == nil {
```

Comment:
We're already doing `if params != nil {` below (not visible) so it's redundant. Should you fix the one below in that case?

Reply:
Yeah good point. Unfortunately the other one would still result in a panic because of the check on `params.Account` immediately afterwards, so I've taken that one out in favor of my addition.